bucket/ddc_bucket/node/store.rs

```diff
@@ -4,12 +4,16 @@ use ink_storage::{collections::Vec as InkVec, traits};
 
 use crate::ddc_bucket::node::entity::Resource;
 use crate::ddc_bucket::{AccountId, Balance, Error::*, Result};
+use ink_storage::collections::HashMap;
 
 use super::entity::{Node, NodeId, NodeTag};
 
 #[derive(traits::SpreadLayout, Default)]
 #[cfg_attr(feature = "std", derive(traits::StorageLayout, Debug))]
-pub struct NodeStore(pub InkVec<Node>);
+pub struct NodeStore {
+    pub store: HashMap<AccountId, NodeId>,
+    pub keys: InkVec<Node>,
```

Comment:
Why do we name a field that stores a vector of nodes (`Vec<Node>`) as `keys` ? To my mind, it's a bit misleading. I would use smth. like `nodes` or `items` instead of `keys`.
As for the `store` field, I would go with `account_node` name for now

Reply:
done